docs/source/notebooks/finch-usage.ipynb

```diff
@@ -59,7 +59,7 @@
       "Help on method frost_days in module birdy.client.base:\n",
       "\n",
       "frost_days(tasmin=None, thresh='0 degC', freq='YS', check_missing='any', missing_options=None, cf_compliance='warn', data_validation='raise', variable=None) method of birdy.client.base.WPSClient instance\n",
-      "    Number of days where daily minimum temperatures are below 0.\n",
+      "    Number of days where daily minimum temperatures are below a threshold temperature.\n",
```

Comment:
This change breaks Jenkins see http://jenkins.ouranos.ca/job/PAVICS-e2e-workflow-tests/job/master/1309/console.

Do we need a new Finch to be deployed to the server?

Else add `# NBVAL_IGNORE_OUTPUT` and we accept the Finch deployed in prod do not match our latest notebooks that is autodeployed to prod.

Reply:
Yes, we should release Finch.

Reply:
> Yes, we should release Finch.

Can we do this soon?  Jenkins broken daily in prod.

Reply:
On it.

Reply:
https://github.com/bird-house/birdhouse-deploy/pull/223